mil_common/utils/mil_tools/scripts/mil-preflight/README.md

```diff
@@ -0,0 +1,23 @@
+# Preflight- Autonomous Robot Verification
```

Comment:
Can you add this document into our docs? You can do that either by just linking to this existing file in the software docs TOC, or you can move this file into the `docs` folder and then treat it just like a normal doc page. Up to you! See `docs/software/index.rst` for how to link it (just copy what is there).

Reply:
Done! Linked the preflight markdown file to the docs.